src/components/Todo.jsx

```diff
@@ -0,0 +1,260 @@
+import { useState } from 'react'
+
+const Todo = () => {
+
+    // states
+    const [tasksState, setTaskState] = useState([])
+    const [taskTitleState, setTaskTitleState] = useState("")
+    const [todoListName, setTodoListName] = useState("My Todo List")
+    const [todoListNameHolder, setTodoListNameHolder] = useState("")
+    const [todoListTasksAmount, setTodoListTasksAmount] = useState("")
+    const [isFiltered, setIsFiltered] = useState(false)
+    const [preFilteredList, setPreFilteredList] = useState([])
+
+    // new task template prototype
+    function Task(title) {
```

Comment:
You won't need to set up classes like this in functional components - what are you trying to achieve here so we can think about how to change it?